scripts/e2e-email/localeQuirks.js

```diff
@@ -63,21 +90,31 @@ var translationQuirks = {
     'sv',
   ],
 
-  'A new device is now syncing to your Firefox Account': [
+  'New sign-in to Sync': [
     'en',
     'en-GB',
     'ca',
     'cy',
+    'es',
     'es-AR',
+    'et',
     'eu',
+    'fa',
     'ff',
+    'fy',
     'he',
+    'hu',
     'id',
     'ko',
     'lt',
-    'pl',
     'nb-NO',
+    'pa',
+    'pl',
+    'sq',
+    'sr',
+    'sr-LATN',
     'sv',
+    'tr',
   ],
```

Comment:
N.B., the green '+' lines are the locales we (temporarily) untranslated in changing the subject.